src/main/java/com/autotune/Autotune.java

```diff
@@ -121,27 +123,45 @@ public static void main(String[] args) {
             LOGGER.info("ROS enabled : {}" ,KruizeDeploymentInfo.is_ros_enabled);
             // Read and execute the DDLs here
             executeDDLs(AnalyzerConstants.ROS_DDL_SQL);
-            if (KruizeDeploymentInfo.local == true) {
-                LOGGER.info("Now running kruize local DDL's ");
-                executeDDLs(AnalyzerConstants.KRUIZE_LOCAL_DDL_SQL);
-                // load available datasources from db
-                loadDataSourcesFromDB();
-                // setting up DataSources
+
+            LOGGER.info("Now running kruize local DDL's ");
```

Comment:
Can we audit all the LOGGER.info's that this PR is adding to make sure that we are not spamming the log during regular operation. One time Logs are ok but they need to have a context associated. Eg "Now running kruize local DDL's" is very vague.